Snowflake.Data.Tests/UnitTests/SFFileTransferAgentTests.cs

```diff
@@ -0,0 +1,551 @@
+            t_locationStage = $"mock-customer-stage_{threadSuffix}";
+
+            // Set values for members that depend on thread variables
+            _srcLocations = new List<string>()
+            {
+                t_realSourceFilePath
+            };
+            _putQuery = "PUT file://" + t_realSourceFilePath + " @DB.SCHEMA.%MOCKTABLE;";
+            _location = Path.GetFullPath(t_locationStage + "/" + LocationId + "/" + LocationPath);
+
+            _responseData = new PutGetResponseData()
+            {
+                autoCompress = false,
+                encryptionMaterial = _encryptionMaterial,
+                localLocation = t_localLocation,
+                overwrite = false,
+                parallel = Parallel,
+                presignedUrl = null,
+                presignedUrls = null,
+                queryId = null,
+                rowSet = null,
+                rowType = null,
+                sourceCompression = AutoDetect,
+                sqlState = null,
+                src_locations = _srcLocations,
+                stageInfo = new PutGetStageInfo()
+                {
+                    location = _location,
+                    locationType = SFRemoteStorageUtil.LOCAL_FS, // Use local storage for testing
+                    path = LocationPath,
+                    presignedUrl = null,
+                    stageCredentials = null
+                },
+                statementTypeId = 0,
+                threshold = 209715200 // Server default threshold
+            };
+
+            _cancellationToken = new CancellationToken();
+
+            _session = new SFSession(ConnectionStringMock, null);
+
+            // Upload setup
+            // Write mock file to upload
+            File.WriteAllText(_srcLocations[0], FileContent);
+            _sourceFileSize = new FileInfo(_srcLocations[0]).Length;
+
+            // Download setup
+            // Write mock file in the local location to download
+            if (!Directory.Exists(_location))
+            {
+                Directory.CreateDirectory(_location);
+            }
+            File.WriteAllText(_location + t_realSourceFilePath, FileContent);
+        }
+
+        [TearDown]
+        public void AfterTest()
+        {
+            // Delete stage directory recursively
+            if (Directory.Exists(t_locationStage))
+            {
+                Directory.Delete(t_locationStage, true);
+            }
+
+            // Upload teardown
+            // Delete mock files
+            foreach (string location in _srcLocations)
+            {
+                File.Delete(location);
+            }
+
+            // Download teardown
+            // Delete local directory recursively
+            if (Directory.Exists(t_localLocation))
+            {
+                Directory.Delete(t_localLocation, true);
+            }
+        }
+
+        [Test]
+        [Ignore("FileTransferAgentTest")]
+        public void FileTransferAgentTestDone()
```

Comment:
We don't need this since https://github.com/snowflakedb/snowflake-connector-net/pull/776

Reply:
Ok, I'll remove it